salt/metalk8s/addons/alert-logger/deployed/deployment.sls

```diff
@@ -0,0 +1,53 @@
+#!jinja | metalk8s_kubernetes
+
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{%- raw %}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metalk8s-alert-logger
+  namespace: metalk8s-monitoring
+  labels:
+    app: metalk8s-alert-logger
+    heritage: metalk8s
+    app.kubernetes.io/name: metalk8s-alert-logger
+    app.kubernetes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
```

Comment:
I see no `spec.replicas`, so we will only ever get one of those loggers (except when the Deployment is doing some rolling update)? Would it make sense to make it configurable? IIRC, you said in the design that since AM will retry sending events to the webhook, I guess it's fine to temporarily lose an instance of this Pod, I just wanted to confirm.

Reply:
Yes, there is only one instance.
It was done on purpose, I don't see the point of having multiple instances of this Pod (I may be wrong).
We can still make it configurable if we want, it should not be a problem.